components/plurals/tests/fixtures/mod.rs

```diff
@@ -65,7 +65,7 @@ pub enum PluralOperandsInput {
 impl From<PluralOperandsInput> for PluralOperands {
     fn from(input: PluralOperandsInput) -> Self {
         match input {
-            PluralOperandsInput::List(operands) => Self {
```

Comment:
It's because the code got touched in intermediate commits that got undone.

Personally I prefer the explicit one for reading, though I don't like typing it so I also use Self quite a bit.